src/client/components/Meal.css

```diff
@@ -0,0 +1,19 @@
+.meal-card {
+    border: 1px solid #ccc;
+    border-radius: 8px;
+    padding: 16px;
```

Comment:
Since you are pretty advanced already, I would suggest you start practicing with the `rem` unit as well :wink: 

It works better across different screen sizes and devices.

https://www.sitepoint.com/understanding-and-using-rem-units-in-css/